charts/pvc-autoresizer/templates/controller/deployment.yaml

```diff
@@ -14,6 +14,9 @@ spec:
     metadata:
       labels:
         app.kubernetes.io/name: pvc-autoresizer
+        {{- if .Values.controller.extraPodLabels}}
```

Comment:
`with` command is more suitable for this.
https://github.com/topolvm/topolvm/pull/743/files#diff-0a5f577850bf4875b5d1f025b1c9d33b199890267147944bf5c5aa62073d7d06R10-R12